__tests__/team.test.ts

```diff
@@ -1,28 +1,28 @@
 import Bot from '../lib'
 import config from './tests.config'
-import {ListTeamMembershipsResult, TeamRole, TeamRolePlural} from '../lib/team-client/types'
+import {TeamRole, TeamDetails} from '../lib/types/keybase1'
 
 test('Team methods with an uninitialized bot', (): void => {
   const alice1 = new Bot()
   const options = {team: config.teams.alicesPlayground.teamname}
   expect(alice1.team.listTeamMemberships(options)).rejects.toThrowError()
 })
 
-function pluralizeRole(r: TeamRole): TeamRolePlural {
+function pluralizeRole(r: TeamRole): 'owners' | 'admins' | 'readers' | 'writers' {
```

Comment:
don't have to change this but as an FYI you should be able to 
1. Mostly infer most return types
1. Use `as const` to stop TS from type widening. Aka it'll see `return 'owners' as const` as the literal `'owners'` and not widen the type to `string`

Reply:
As long as I'm sticking with explicit return types everywhere (currently using that setting), the `as const` thing is unnecessary, right? I don't think this widens to `string` as it is.